app/forms/registration/early_years_experiences_form.rb

```diff
@@ -0,0 +1,18 @@
+module Registration
+  class EarlyYearsExperiencesForm < BaseForm
+    attr_accessor :early_years_experience
+
+    validates :early_years_experience, presence: true
+
+    def early_years_experiences
+      YAML.load_file(Rails.root.join('data/early_years_experience.yml'))
```

Comment:
@jack-coggin look in the `Trainee` namespace and follow that pattern.

Reply:
[958767a](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/1048/commits/958767ada9f7b9f4fd71eb09d643262d833c08ee)